### PR TITLE
[Merged by Bors] - doc(Computability/Halting): Mark named theorems

### PR DESCRIPTION
**Pull request by hmonroe**
Mark these named theorems: Rice, Downward and Upward Skolem

### CODE REVIEW

File: Mathlib/Computability/Halting.lean [hunk 1]
```diff
@@ -194,6 +194,7 @@ theorem to_re {p : α → Prop} (hp : ComputablePred p) : RePred p := by
   cases a; cases f n <;> simp
 #align computable_pred.to_re ComputablePred.to_re
 
+/- **Rice's Theorem** -/
```
Comment (alexjbest) [inline Mathlib/Computability/Halting.lean:197]:
```suggestion
/-- **Rice's Theorem** -/
```
only `/--` comments show up in the docs

**Comment by alexjbest** [comment]
please tag as `awaiting-review` when ready

**Comment by alexjbest** [comment]
maintainer merge

**Comment by kmill** [comment]
Thanks!

bors r+

**Pull request closed without merging**